assignment6.txt

```diff
@@ -0,0 +1,34 @@
+/*Rebecca Hale Assignment 6 Part 2*/
+1.  There are problems with the code.  First, num is not given a value 
```

Comment:
Right! so to fix this you could just move the array declaration to after input is scanned.